packages/botonic-nlu/package.json

```diff
@@ -24,7 +24,7 @@
     "fs": "0.0.1-security",
     "inquirer": "^7.0.4",
     "langs": "^2.0.0",
-    "sqlite-async": "^1.0.11"
+    "sqlite-async": "1.0.11"
```

Comment:
I understood from the PR description that it was updating this dependency not freezing it

Reply:
I've tried to update to `1.1.0` as suggested by dependabot with no success, so I have rolled back to working version. In the same  process I've updated `tfjs` dependencies